integration/build_test.go

```diff
@@ -126,129 +130,9 @@ func testBuild(t *testing.T, context spec.G, it spec.S) {
-				Execute(name, source)
-			Expect(err).NotTo(HaveOccurred(), logs.String())
-
-			container, err = docker.Container.Run.
-				WithEnv(map[string]string{"PORT": "8080"}).
-				WithPublish("8080").
-				WithPublishAll().
-				Execute(image.ID)
-			Expect(err).NotTo(HaveOccurred())
-
-			Eventually(container).Should(BeAvailable())
-
-			response, err := http.Get(fmt.Sprintf("http://localhost:%s", container.HostPort("8080")))
-			Expect(err).NotTo(HaveOccurred())
-			defer response.Body.Close()
-
-			Expect(response.StatusCode).To(Equal(http.StatusOK))
-
-			content, err := ioutil.ReadAll(response.Body)
-			Expect(err).NotTo(HaveOccurred())
-			Expect(string(content)).To(Equal("Hello world!"))
-
-			Expect(logs).To(ContainLines(ContainSubstring("Go Distribution Buildpack")))
-			Expect(logs).To(ContainLines(ContainSubstring("Go Build Buildpack")))
-
-			Expect(logs).NotTo(ContainLines(ContainSubstring("Go Mod Vendor Buildpack")))
-			Expect(logs).NotTo(ContainLines(ContainSubstring("Dep Buildpack")))
-		})
-	})
-
-	context("when building a dep app that is vendored", func() {
-		var (
-			image     occam.Image
-			container occam.Container
-
-			name   string
-			source string
-		)
-
-		it.Before(func() {
-			var err error
-			name, err = occam.RandomName()
-			Expect(err).NotTo(HaveOccurred())
-
-			source, err = occam.Source(filepath.Join("testdata", "dep_vendored"))
-			Expect(err).NotTo(HaveOccurred())
-		})
-
-		it.After(func() {
-			Expect(docker.Container.Remove.Execute(container.ID)).To(Succeed())
-			Expect(docker.Image.Remove.Execute(image.ID)).To(Succeed())
-			Expect(docker.Volume.Remove.Execute(occam.CacheVolumeNames(name))).To(Succeed())
-			Expect(os.RemoveAll(source)).To(Succeed())
-		})
-
-		it("creates a working OCI image", func() {
-			var err error
-			var logs fmt.Stringer
-			image, logs, err = pack.WithNoColor().Build.
-				WithBuildpacks(goBuildpack).
-				WithPullPolicy("never").
-				Execute(name, source)
-			Expect(err).NotTo(HaveOccurred(), logs.String())
-
-			container, err = docker.Container.Run.
-				WithEnv(map[string]string{"PORT": "8080"}).
-				WithPublish("8080").
-				WithPublishAll().
-				Execute(image.ID)
-			Expect(err).NotTo(HaveOccurred())
-
-			Eventually(container).Should(BeAvailable())
-
-			response, err := http.Get(fmt.Sprintf("http://localhost:%s", container.HostPort("8080")))
-			Expect(err).NotTo(HaveOccurred())
-			defer response.Body.Close()
-
-			Expect(response.StatusCode).To(Equal(http.StatusOK))
-
-			content, err := ioutil.ReadAll(response.Body)
-			Expect(err).NotTo(HaveOccurred())
-			Expect(string(content)).To(ContainSubstring("Hello, World!"))
-
-			Expect(logs).To(ContainLines(ContainSubstring("Go Distribution Buildpack")))
-			Expect(logs).To(ContainLines(ContainSubstring("Go Build Buildpack")))
-
-			Expect(logs).NotTo(ContainLines(ContainSubstring("Go Mod Vendor Buildpack")))
-			Expect(logs).NotTo(ContainLines(ContainSubstring("Dep Buildpack")))
-		})
-	})
```

Comment:
I think I am going to be a little bit stubborn here ask why you moved these. To my understanding they are all testing the same group order, which a go build with no package managers. I don't know that it makes sense to move these tests in to the tests that use those package managers in the group because these tests aren't triggering those groups.

Reply:
fair point. works for me.